maintainers/maintainer-list.nix

```diff
@@ -4121,6 +4121,12 @@
     githubId = 12491746;
     name = "Masato Yonekawa";
   };
+  hyzual = {
+    email = "hyzual@gmail.com";
+    github = "Hyzual";
+    githubId = 2051507;
+    name = "Joris Masson";
+  };
```

Comment:
Adding maintainers is usually in another commit `maintainers: add Joris Masson`

Reply:
Oops sorry about that 😅. I'll split into two commits. Thanks for the review 😊